action.yml

```diff
@@ -288,6 +288,13 @@ runs:
         fi
         echo ::endgroup::
 
+    - name: Show env
+      shell: lispsh -eo pipefail {0}
+      run: |
+        set -e
+        pwd
+        ls -l
+        env | sort
```

Comment:
Would this lead to pipeline secrets getting accidentally leaked?

Reply:
No GitHub is smart enough and replaces secrets with **********.

Anyway, this commands are for debug output which will be removed in the final version if the PR.